### ISSUE TITLE
If more than one user tries to use it at the same time, the MIDI files will be overwritten. I can fix this later somewhat easily.

### ISSUE THREAD

**Issue by sagnew** [i/Vireo#6]
We will have to create a new MIDI file after every time the app is used, and then delete it after the user leaves the site. We can have several midi files with different names existing on the server at a time, and the name of the file will be returned when the HTML template is rendered.

**Comment by kbambridge** [comment]
Yeah dude this was the most fucking hackey implementation ever haha
